builtin/credential/cert/path_certs.go

```diff
@@ -228,6 +234,23 @@ func (b *backend) pathCertWrite(ctx context.Context, req *logical.Request, d *fr
 		}
 	}
 
+	var parsedCIDRs []*sockaddr.SockAddrMarshaler
+	if boundCIDRListRaw, ok := d.GetOk("bound_cidrs"); ok {
```

Comment:
This pattern is usually only required for paths that handle both create and update operations (have an existence check).  I think you could simplify this by just iterating of the the list without checking if the parameter is set.

Reply:
:+1: